### ISSUE TITLE
Improve accessibility of the website

### ISSUE THREAD

**Issue by agazso** [diete-design/diete.design#225]
There is a tool for detecting accessibility issues: https://wave.webaim.org/report#/https://diete.design

Currently it reports 21 errors, 9 contrast errors and 3 alerts.

Among the errors there are low hanging fruits, like missing title and incorrect form labels and empty buttons.

The contrast errors are due to the `Code` component using a small font. This could be fixed by using a 16px font which is also what the design proposes.

I also added a new label `accessibility` to mark accessibility related issues.
